ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/EVPN_MULTICAST_TESTS.yml

```diff
@@ -362,6 +362,8 @@ tenants:
             ip_address_virtual: 10.2.23.1/24
             # test for fast-leave when igmp_snooping_querier.fast_leave: true but evpn_l2_multicast.enabled is not set
             # fast-leave is not expected for VLAN 230 since evpn_l2_multicast.enabled is not set
+            # TODO: Revisit this test. I don't believe this should be the behavior.
+            # evpn_l2_multicast.enabled should not be required for fast-leave configuration since it's not OISM specific.
```

Comment:
you are changing that behavior in this PR, so this is not a TODO?

Reply:
correct, TODO within this PR, reminder for myself :) Wanted to have a discussion first before I continued working on this PR!